test/functional/legacy/105_filename_modifiers_spec.lua

```diff
@@ -45,6 +45,7 @@ describe('filename modifiers', function()
     execute([=[Put fnamemodify("abc\ndef",       ':S'      )]=])
     execute([=[set shell=tcsh]=])
     execute([=[Put fnamemodify("abc\ndef",       ':S'      )]=])
+    execute([=[Put expand('%:r:S') == shellescape(expand('%:r'))]=])
```

Comment:
This line appears to be breaking the test harness.  I get this lua traceback that I can't make any sense out of.

```
Error → test/functional/legacy/105_filename_modifiers_spec.lua @ 10
filename modifiers is working
...m/neovim/.deps/usr/share/lua/5.1/nvim/msgpack_stream.lua:65: end of file

stack traceback:
        ...m/neovim/.deps/usr/share/lua/5.1/nvim/msgpack_stream.lua:65: in function 'run'
        ...om/neovim/.deps/usr/share/lua/5.1/nvim/async_session.lua:43: in function 'run'
        ...thub.com/neovim/.deps/usr/share/lua/5.1/nvim/session.lua:160: in function '_blocking_request'
        ...thub.com/neovim/.deps/usr/share/lua/5.1/nvim/session.lua:83: in function 'request'
        ./test/functional/helpers.lua:66: in function 'nvim'
        ./test/functional/helpers.lua:307: in function 'curbuf'
        ./test/functional/helpers.lua:323: in function 'curbuf_contents'
        ./test/functional/helpers.lua:343: in function 'expect'
        ...
        ...thub.com/neovim/.deps/usr/share/lua/5.1/busted/block.lua:150: in function 'execute'
        ...ithub.com/neovim/.deps/usr/share/lua/5.1/busted/init.lua:7: in function 'executor'
        ...ithub.com/neovim/.deps/usr/share/lua/5.1/busted/core.lua:276: in function <...ithub.com/neovim/.deps/usr/share/lua/5.1/busted/core.lua:276>
```
The commands themselves work fine when run manually, so it seems to be something specific to the testing.